src/products/components/ProductVideoUrlDialog/ProductVideoUrlDialog.tsx

```diff
@@ -0,0 +1,85 @@
+import Button from "@material-ui/core/Button";
+import Dialog from "@material-ui/core/Dialog";
+import DialogActions from "@material-ui/core/DialogActions";
+import DialogContent from "@material-ui/core/DialogContent";
+import DialogTitle from "@material-ui/core/DialogTitle";
+import TextField from "@material-ui/core/TextField";
+import Typography from "@material-ui/core/Typography";
+import Form from "@saleor/components/Form";
+import FormSpacer from "@saleor/components/FormSpacer";
+import { buttonMessages } from "@saleor/intl";
+import { ProductDetails_product } from "@saleor/products/types/ProductDetails";
+import React from "react";
+import { FormattedMessage } from "react-intl";
+
+interface ProductVideoUrlDialogProps {
+  product: ProductDetails_product;
+  open: boolean;
+  onClose();
+  onSubmit(videoUrl: string);
```

Comment:
Code consistency 🤷‍♂️

```suggestion
  onClose: () => void;
  onSubmit: (videoUrl: string) => void;
```